src/@next/components/atoms/DropdownMenu/test.tsx

```diff
@@ -0,0 +1,106 @@
+import { mount, shallow } from "enzyme";
+import "jest-styled-components";
+import React from "react";
+
+import { DropdownMenu } from ".";
+
+describe("<DropdownMenu />", () => {
+  const content = [
+    {
+      content: <span>This is test</span>,
+      onClick: jest.fn(),
+    },
+    {
+      content: <span>This is test 2</span>,
+      onClick: jest.fn(),
+    },
+    {
+      content: <span>This is test 3</span>,
+      onClick: jest.fn(),
+    },
+  ];
+  const header = <p>press me</p>;
+
+  const DEFAULT_PROPS = {
+    content,
+    header,
+  };
+  it("exists", () => {
+    const wrapper = shallow(
+      <DropdownMenu type="clickable" {...DEFAULT_PROPS} />
+    );
+
+    expect(wrapper.exists()).toEqual(true);
+  });
+  it("should not contain any list items if not clicked/hovered", () => {
+    const wrapperClickable = shallow(
+      <DropdownMenu type="clickable" {...DEFAULT_PROPS} />
+    );
+    expect(wrapperClickable.find("li").length).toEqual(0);
+
+    const wrapperHoverable = shallow(
+      <DropdownMenu type="clickable" {...DEFAULT_PROPS} />
+    );
+    expect(wrapperHoverable.find("li").length).toEqual(0);
```

Comment:
I think it might be better to use this pattern for code formatting: http://wiki.c2.com/?ArrangeActAssert

Reply:
Changed to mentioned formatting pattern